app/controllers/admin/reports/system-logs/activity-logs.js

```diff
@@ -1,23 +1,45 @@
 import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
 
-export default Controller.extend({
-  columns: [
-    {
-      propertyName     : 'actor',
-      title            : 'Actor',
-      disableSorting   : true,
-      disableFiltering : true
-    },
-    {
-      propertyName : 'time',
-      template     : 'components/ui-table/cell/admin/reports/system-logs/activity-logs/cell-time',
-      title        : 'Time'
-    },
-    {
-      propertyName     : 'action',
-      title            : 'Actions',
-      disableSorting   : true,
-      disableFiltering : true
-    }
-  ]
-});
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
+  per_page = 100;
+  sort_by = 'time';
+  sort_dir = 'ASC';
+
+  @computed()
+  get columns() {
+    return [
+      {
+        name            : 'Actor',
+        valuePath       : 'actor',
+        headerComponent : 'tables/headers/sort',
+        isSortable      : true
+      },
+      {
+        name            : 'Time',
+        valuePath       : 'time',
+        cellComponent   : 'ui-table/cell/admin/reports/system-logs/activity-logs/cell-time',
+        headerComponent : 'tables/headers/sort',
+        isSortable      : true
+
+      },
+      {
+        name      : 'Actions',
+        valuePath : 'action'
+      }];
+  }
+
+  @computed('model.[]')
+  get rows() {
```

Comment:
Use model directly. One less computed property. One less iteration of the dataset.